batavia/modules/time.js

```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
     }
 
     var date
-    if (arguments.length === 1) {
+    if (arguments.length === 1)
+    {
         // catching bad types
-        if (types.isinstance(seconds, [types.Complex])) {
+        if (types.isinstance(seconds, [types.Complex]))
+        {
             throw new exceptions.TypeError.$pyclass("can't convert " + type_name(seconds) + ' to int')
-        } else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) {
+        } 
+        else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) 
+        {
             throw new exceptions.TypeError.$pyclass('an integer is required (got type ' + type_name(seconds) + ')')
         }
 
         date = new Date(seconds * 1000)
```

Comment:
At column 85: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)